app/src/main/java/io/gnosis/safe/ui/safe/settings/app/GetInTouchFragment.kt

```diff
@@ -0,0 +1,119 @@
+import io.gnosis.safe.di.components.ViewComponent
+import io.gnosis.safe.ui.base.BaseFragment
+import pm.gnosis.svalinn.common.utils.openUrl
+import pm.gnosis.svalinn.common.utils.snackbar
+import timber.log.Timber
+
+class GetInTouchFragment : BaseFragment<FragmentGetInTouchBinding>() {
+
+    override fun inject(component: ViewComponent) {
+        component.inject(this)
+    }
+
+    override fun inflateBinding(inflater: LayoutInflater, container: ViewGroup?): FragmentGetInTouchBinding =
+        FragmentGetInTouchBinding.inflate(inflater, container, false)
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setHasOptionsMenu(true)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        (activity as AppCompatActivity).setSupportActionBar(binding.getInTouchToolbar)
+        (activity as AppCompatActivity).supportActionBar?.setDisplayHomeAsUpEnabled(true)
+
+        with(binding) {
+            email.setOnClickListener {
+                sendEmail()
+            }
+            discord.setOnClickListener {
+                openDiscord()
+            }
+            twitter.setOnClickListener {
+                openTwitter()
+            }
+            helpCenter.setOnClickListener {
+                openHelpCenter()
+            }
+            featureSuggestion.setOnClickListener {
+                openFeatureSuggestionPage()
+            }
+        }
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        when (item.itemId) {
+            android.R.id.home -> {
+                findNavController().navigateUp()
+                return true
+            }
+        }
+        return super.onOptionsItemSelected(item)
+    }
+
+    private fun openDiscord() {
+        startActivity(Intent(Intent.ACTION_VIEW, Uri.parse("https://discord.gg/${getString(R.string.id_discord)}")))
+    }
+
+    private fun openTwitter() {
+        val intent = try {
+            // get the Twitter app if possible
+            requireContext().packageManager.getPackageInfo("com.twitter.android", 0)
+            Intent(Intent.ACTION_VIEW, Uri.parse("twitter://user?screen_name=${getString(R.string.id_twitter)}")).apply {
+                addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+            }
+        } catch (e: java.lang.Exception) {
+            // no Twitter app, revert to browser
+            Intent(Intent.ACTION_VIEW, Uri.parse("https://twitter.com/${getString(R.string.id_twitter)}"))
+        }
+        startActivity(intent)
+    }
+
+    private fun openHelpCenter() {
+        requireContext().openUrl(getString(R.string.link_help_center))
+    }
+
+    private fun openFeatureSuggestionPage() {
+        requireContext().openUrl(getString(R.string.link_feature_suggestion))
+    }
+
+    // Do we need telegram channel or is discord sufficient?
```

Comment:
This question should be asked in the issue of the epic. If we leave in code we just forget about it and probably won't address.

Reply:
asked & removed comment. But I would leave code for opening telegram app in case we need it later